lib/googleauth/credentials.rb

```diff
@@ -40,10 +40,55 @@ module Auth
     class Credentials
       TOKEN_CREDENTIAL_URI = "https://oauth2.googleapis.com/token".freeze
       AUDIENCE = "https://oauth2.googleapis.com/token".freeze
-      SCOPE = [].freeze
-      PATH_ENV_VARS = [].freeze
-      JSON_ENV_VARS = [].freeze
-      DEFAULT_PATHS = [].freeze
+
```

Comment:
Don't these methods need documentation?

Reply:
They do, but I didn't want to go through the effort if the approach was a non-starter.